finish/action.yml

```diff
@@ -20,17 +20,17 @@ runs:
     # Try to get pr artifact from current workflow
     - name: Download current PR artifact
       id: get-pr-artifact
-      uses: actions/download-artifact@v3
+      uses: actions/download-artifact@v4
       continue-on-error: true
       with:
-        name: pr
+        name: pr-id
 
     # If the above fails, try to get the latest pr artifact from the PR related workflows
     - name: Download latest PR artifact
       uses: dawidd6/action-download-artifact@v6
       if: ${{ steps.get-pr-artifact.outcome == 'failure' }}
       with:
-        name: pr
+        name: pr-id
         pr: ${{ github.event.pull_request.number }}
         # Can be ID or workflow file name, if empty falls back to the latest successful run of the current workflow
         workflow: ${{ env.PR_ARTIFACT_WORKFLOW }}
```

Comment:
**issue**: (this is the closest line I can comment): the step with the name "Load the Ephemeral Instance URL" relies on the `ls-preview-url.txt` file being downloaded. The changes in this PR, more specifically [this change](https://github.com/localstack/setup-localstack/pull/43/files#diff-73902b4047d164cbf4ac410682b93d1a3de5eff8a5a4e7794e2e765e997b07acL107), will break this functionality, as it uploads the file into a new separate artifact, which is not downloaded in this action.

Reply:
**follow-up**: introducing the same download steps as for the `pr-id` here as well, so we can use the instance url